airflow/providers/celery/executors/celery_executor_utils.py

```diff
@@ -155,7 +155,7 @@ def _execute_in_fork(command_to_exec: CommandType, celery_task_id: str | None =
         args.func(args)
         ret = 0
     except Exception as e:
-        log.exception("[%s] Failed to execute task %s.", celery_task_id, str(e))
+        log.exception("[%s] Failed to execute task %s.", celery_task_id, e)
```

Comment:
Probably not belong in this PR but this second argument really isn’t needed since `exception` would log this automatically.